File: multiplesOf3And5/multiplesOf3And5.cpp
#include <iostream>
using namespace std;
int sum = 0;

int main(){

    for (int i=0; i<1000; i++)
    {
        if (i%3==0 || i%5==0){
            sum = sum + i;
        }
    }
    cout<<sum<<endl;

}

File: js/1-multiplesOf3And5/multiplesOf3And5.js
let i;
let sum = 0;
function multiplesOf3And5(){
    for (i = 1; i < 1000; i++){
    if (i % 3 === 0 || i % 5 === 0){
        sum = sum + i;
        }
    }
    return sum;
}

console.log(multiplesOf3And5());
